src/analytics.js

```diff
@@ -142,6 +144,10 @@ const getMetrics = ( extra = {} ) => ( {
 	elapsed: elapsed + ( Date.now() - start ),
 	scrollDepthMax,
 	scrollDepthNow,
+	hour: new Date().getHours(),
+	day: new Date().getDay() + 1,
```

Comment:
Month I get, but why offset this by 1?

Reply:
Good q, I figured if I did it for month I should do it for day too, otherwise it's a little inconsistent and we're exposing the idea of 0 indexed lists to users. So Sunday is 1, etc, etc